### DISCUSSION TITLE
Dns01Authorization Rate limit for '/acme' reached

### DISCUSSION THREAD

**Discussion by kevin-dfo** [shibayan/appservice-acmebot#313] [Q&A] [answered]
Hi,

I got an error Dns01Authorization Rate limit for '/acme' reached when it tried to renew the certificate. 

Will it try to renew the cert again at a later date or will I need to do something?

**Comment by shibayan** [comment, marked as answer]
Let's Encrypt has a rate limit, so when that limit is reached, you will need to wait a while for it to be lifted.
https://letsencrypt.org/docs/rate-limits/

**Reply by shibayan** [reply]
Fixed v3.4.4
